src/main/java/nl/hsac/fitnesse/junit/JUnitAllureFrameworkListener.java

```diff
@@ -0,0 +1,266 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by Fernflower decompiler)
+//
```

Comment:
I accidentally deleted the listener without committing. Only had the class file in the target folder, so I decompiled and refactored it back to what it was :)